src/Cmfd.cpp

```diff
@@ -498,8 +475,8 @@ void Cmfd::computeDs(int moc_iteration) {
             d_tilde = 0.0;
           else
             d_tilde =
-                _materials[cell]->getDifTilde()[surface*_num_cmfd_groups + e] *
-                (1 - _relax_factor) + _relax_factor * d_tilde;
+              _materials[cell]->getDifTilde()[surface*_num_cmfd_groups + e] *
+              (1 - _relax_factor) + _relax_factor * d_tilde;
```

Comment:
The line is was indented 4 spaces and I just changed the indent back to 2 spaces.